libs/client/package.json

```diff
@@ -8,6 +8,7 @@
     "@tskmgr/common": "0.0.0",
     "uuid": "^9.0.1",
     "@nx/devkit": "18.0.7",
-    "form-data": "2.3.3"
+    "form-data": "2.3.3",
+    "nx": "18.0.7"
```

Comment:
can we discuss it ? technically tskmgr should not be coupled to Nx.

Reply:
this comes from [lient-example.ts](https://github.com/mathpaquette/tskmgr/blob/master/libs/client/src/lib/client-example.ts). The example should be separated from client library, then the lib will become uncoupled from NX.